services/audit-service/src/main/java/datawave/microservice/audit/replay/util/LockableCacheInspector.java

```diff
@@ -0,0 +1,22 @@
+package datawave.microservice.audit.replay.util;
```

Comment:
This doesn't seem audit-specific to me. Why isn't it in hazelcast-client?

Reply:
I have no objection to that.  Do you think it should still be a separate interface, or should I just add the methods to the CacheInspector interface?

Reply:
I'd keep it separate. We don't want to require that every cache type supports locking functions.

Reply:
ok, cool.  that's how it is in the latest commit.